platform-api/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java

```diff
@@ -213,27 +240,42 @@ public UsersWorkspaceDto startTemporary(WorkspaceConfigDto cfg, @QueryParam("acc
     @POST
     @Path("/{id}/runtime")
     @Produces(APPLICATION_JSON)
+    @RolesAllowed("user")
     public UsersWorkspaceDto startById(@PathParam("id") String workspaceId,
                                        @QueryParam("environment") String envName,
                                        @QueryParam("accountId") String accountId)
             throws ServerException, BadRequestException, NotFoundException, ForbiddenException {
+        ensureUserIsWorkspaceOwner(workspaceId);
+
+        final Map<String, String> params = ImmutableMap.of("accountId", accountId, "workspaceId", workspaceId);
+        permissionManager.checkPermission(START_WORKSPACE, getCurrentUserId(), params);
```

Comment:
Impossible.
When it is in one line it exceeds _right margin_ __140 chars__.

What i can do is use multi-line `ImmutableMap.of` declaration:

For instance:
```java
checkPermission(START_WORKSPACE, getCurrentUserId(), ImmutableMap.of("accountId", accountId, 
                                                                     "workspaceId", workspaceId));
```

Or wrap `checkPermission` method arguments:

For instance:
```
checkPermission(START_WORKSPACE,
                getCurrentUserId(), 
                ImmutableMap.of("accountId", accountId, "workspaceId", workspaceId));
```

What do you prefer in this case? @skabashnyuk